docs/rules/wrap-iife.md

```diff
@@ -80,3 +86,25 @@ Examples of **correct** code for the `"any"` option:
 var x = (function () { return { y: 1 };}()); // wrapped call expression
 var x = (function () { return { y: 1 };})(); // wrapped function expression
 ```
+
+### functionPrototypeMethods
+
+Examples of **incorrect** code for this rule with the `"inside", { "functionPrototypeMethods": true }` options:
+
+```js
+/* eslint wrap-iife: [2, "inside", { functionPrototypeMethods: true }] */
+
+function(){ foo(); }()
+(function(){ foo(); }())
+function(){ foo(); }.call(bar)
```

Comment:
Lines 1 and 3 of this example have invalid syntax. Could you modify it so that it's valid?

e.g. something like

```js
var x = function(){ foo(); }()
```